### ISSUE TITLE
After set use_zip_chunks to true, it appears as false in the response.

### ISSUE THREAD

**Issue by k1won** [openvinotoolkit/cvat#4493]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour

- When POST /tasks/{id}/data is executed, the value set in the request body and the value appears in the response should be the same.
### Current Behaviour

- Use_zip_chunks, use_cache, and copy_data are all set to true, but it is appears as false in the response.

![image](https://user-images.githubusercontent.com/12894869/159334293-6c11c987-504d-429a-8d61-c3e92797ff03.png)

### Possible Solution

- Modify the return code of upload_finished() method of TaskViewSet class
```diff
diff --git a/cvat/apps/engine/views.py b/cvat/apps/engine/views.py
index cca75503..29d58a1a 100644
--- a/cvat/apps/engine/views.py
+++ b/cvat/apps/engine/views.py
@@ -699,7 +699,7 @@ class TaskViewSet(UploadMixin, viewsets.ModelViewSet):
             if 'stop_frame' not in serializer.validated_data:
                 data['stop_frame'] = None
             task.create(self._object.id, data)
-            return Response(serializer.data, status=status.HTTP_202_ACCEPTED)
+            return Response(data, status=status.HTTP_202_ACCEPTED)
         return Response(data='Unknown upload was finished',
                         status=status.HTTP_400_BAD_REQUEST)
```

### Steps to Reproduce (for bugs)

1. create a task with this api endpoint : POST /api/tasks
1. upload image data with this api endpoint : POST /api/tasks/{id}/data
   Set the following items to true in the request body.
- use_zip_chunks
- use_cache
- copy_data

### Your Environment

- Git hash commit (`git log -1`): dd60b2d8
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS): ubuntu 18.04
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

**Comment by bsekachev** [comment]
@Marishka17 

Can you look into this issue?

**Comment by jevansbio** [comment]
So this is clearly still an issue. Is it just that the response does not reflect the request, or are the actual settings not being used?

**Comment by bsekachev** [comment]
> Is it just that the response does not reflect the request

Yes. Settings are used during creating a task, but looks like something wrong with reading them from the server. Maybe they are even not saved on the server after task creation, because basically they are not used anywhere else.
